### DISCUSSION TITLE
How can I change language strings in login/signup pages?

### DISCUSSION THREAD

**Discussion by Johan-IDX** [wasp-lang/wasp#2171] [Archived]
How can I change the language to spanish in login/signup pages? I don't see any config or documentation about this issue:
![image](https://github.com/wasp-lang/wasp/assets/172074885/5426e9ef-70e9-4ec8-8116-4ef78f7af117)
Thank you

**Comment by Martinsos** [comment]
I am afraid that is not currently possible! You could possibly achieve it in a hacky way, by doing some CSS tricks where you hide existing text and add a new one. But that is not ideal. I opened an issue here https://github.com/wasp-lang/wasp/issues/2174 .
